### ISSUE TITLE
Be smarter about parsing output for `tally`

### ISSUE THREAD

**Issue by seblyng** [seblyng/cargo-aoc#48]
Currently, the way we recognises if a day is implemented or not is if we find the output from the `time` function in the template.

This will fail if anything else is printed to stdout as we assumes it's the first and second line of the output.

I think we can be smarter about this and still recognise the days as implemented even if there is some other prints here and there in the output

**Comment by sivertjoe** [comment]
Yeah,  I thought about checking the exit code, but I must have forgotten. Anyways yeah you're right, just check the exit code of the day to test whether its  implemented or not

**Comment by sivertjoe** [comment]
I belive this is fixed.

This is the code for verifying a day
```rs
 if !res.status.success() {
        let details = extract_runtime_error(res.stderr);
        if details == "not implemented" {
            return Err(Error {
                title: info.title.clone(),
                day,
                r#type: ErrorTypes::NotImplementd,
            });
        }
...
```
